File: cmmdc-cmmmc-euclid/fisier.py
def cmmdc(a, b):
    while b != 0:
        r = a % b
        a = b
        b = r
    return a
def cmmmc(a, b):
    d = cmmdc(a, b)
    return (a * b) / d
a = int(input())
b = int(input())
print(int(cmmdc(a, b)), , int(cmmmc(a, b)))

File: cmmdc-cmmmc-euclid/fisier.cpp
#include <iostream>
using namespace std;
int cmmdc(int a, int b) {
    while (b != 0) {
        int r = a % b;
        a = b;
        b = r;
    } return a;
}
int cmmmc(int a, int b) {
    int d = cmmdc(a, b);
    return (a * b) / d;
}
int main() {
    int a, b;
    cin >> a >> b;
    cout << cmmdc(a, b) << " " << cmmmc(a, b);
    return 0;
}
